src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,20 @@
-package practice;
-
-public class CandidateValidator {
-    //write your code here
-}
+package practice;
+
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int MIN_AGE = 35;
+    private static final String NATIONALITY = "Ukrainian";
+    private static final int YEARS_IN_UKRAINE = 10;
+
+    @Override
+    public boolean test(Candidate candidate) {
+        String[] partsOfYearsInUkraine = candidate.getPeriodsInUkr().split("-");
+        int periodInUkraine = Integer.parseInt(partsOfYearsInUkraine[1])
+                - Integer.parseInt(partsOfYearsInUkraine[0]);

```

Comment:
0, 1 - indexes are magic numbers, should be avoided 
```suggestion
        int periodInUkraine = Integer.parseInt(partsOfYearsInUkraine[1])
                - Integer.parseInt(partsOfYearsInUkraine[0]);

```